tools/dune_rule_gen/coq_rules.ml

```diff
@@ -217,6 +171,11 @@ module Context = struct
 
 end
 
+(* Super hack, to be removed *)
+let fixup_install_locations file =
```

Comment:
This seems very suspicious and should not be needed.

Reply:
I think I figured it out, cf latest commit

Reply:
(I tested dev mode and opam install locally)

Reply:
I guess I did something weird when testing dev mode because now it doesn't work

Reply:
another attempt 74119b4078